src/components/form-components/FormComponents.tsx

```diff
@@ -25,7 +25,7 @@ type ComponentProps = {
 type Props = {
   name: string
   type?: string
-  component: React.ComponentType<ComponentProps> | 'input' | 'textarea' | 'select'
+  component: React.ComponentType<any> | 'input' | 'textarea' | 'select'
```

Comment:
I had to move back to `any`, because some of the Amsterdam input elements do not expose themselves like native input-elements.